clms-angular/src/app/modules/courses/modals/select-modules-dialog/select-modules-dialog.component.ts

```
@@ -0,0 +1,40 @@
+    TableModule,
+    ButtonDirective
+  ],
+  standalone: true
+})
+export class SelectModulesDialogComponent {
+  modules: CourseModuleProjection[] = []; // Input list of modules
+  selectedModules: CourseModuleProjection[] = []; // Array to hold selected modules
+
+  constructor(public ref: DynamicDialogRef, public config: DynamicDialogConfig<SelectModulesDialogData>) {
+    // Load the modules from config if provided
+    if (config.data && config.data.modules) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)